tests/validator20/validate_apis_test.py

```diff
@@ -7,6 +7,7 @@ def test_api_level_params_ok():
     # since they are peers.
     apis = {
         '/tags/{tag-name}': {
+            'x-ignore-me': 'DO NOT LOOK AT ME!',
```

Comment:
Can you pull this out into another unit test? It will help keep the tests easier to maintain if only a single concern is exercised per test. :smile: